modules/@apostrophecms/notification/ui/apos/apps/ApostropheNotification.js

```diff
@@ -0,0 +1,56 @@
+import Vue from 'apostrophe/vue';
+
+export default function() {
+  /* eslint-disable no-new */
+  return new Vue({
+    el: '#apos-notification',
+    data () {
+      return {
+        notifications: []
+      };
+    },
+    computed: {
+      apos() {
+        return window.apos;
+      }
+    },
+    async mounted() {
+      if (apos.scene !== 'apos') {
```

Comment:
This error can never happen, the vue components are only in the bundle when the scene is apos.

Reply:
Done